playlists-prod/excel.yaml

```diff
@@ -1201,6 +1201,18 @@
   group: Preview APIs
   api_set:
     ExcelApi: BETA (PREVIEW ONLY)
+- id: excel-data-types-references
+  name: 'Data types: Entity values with references'
+  fileName: data-types-references.yaml
+  description: >-
+    This sample shows how to create entity values with references to other
+    entity values. An entity value is a container for data, and this container
+    can have references (or links) to other entity values on the same worksheet.
```

Comment:
The "on the same worksheet" is not correct. It is not possible for an entity value to have references to other entities on the worksheet. The references can only be within an entity value.

Reply:
Also, I don't think of "references" link, but rather a pointer to an entity value.